src/main/java/org/broadinstitute/consent/http/models/User.java

```diff
@@ -95,7 +96,7 @@ public User(Integer userId, String email, String displayName, Date createDate,
    * @param json A json string that may or may not be correctly structured as a DACUser
    */
   public User(String json) {
-    Gson gson = new Gson();
+    Gson gson = GsonUtil.gsonBuilderWithAdapters().create();
```

Comment:
I think we should add to the PR description a note about why we need to make so many `Gson` changes in this PR.